jsonrpc/server.go

```diff
@@ -43,7 +43,7 @@ type Error struct {
 	Data    any    `json:"data,omitempty"`
 }
 
-func rpcErr(code int, data any) *Error {
+func RPCErr(code int, data any) *Error {
```

Comment:
maybe rename it just to `Err(...)` ? it's already clear what is doing outside of package json**rpc**.Err(...)